examples/node/scripts/reactions/whoReacted.ts

```diff
@@ -0,0 +1,41 @@
+import {
```

Comment:
all reaction examples should be inside /scripts/publication/reactions folder I think as it is a submodule of Publication